doc/usersguide/index.rst

```diff
@@ -30,6 +30,8 @@ much manual data transfer you wish to do.
 +-------------------------+----------------+----------------+
 |  Tripoli4 Materials     | N              |  None          |
 +-------------------------+----------------+----------------+
+|  OpenMC Materials       | N              |  M             |
```

Comment:
This should be A right? With UWUW the materials are automatically produced

Reply:
Yes! Thanks. Updating.